orchagent/portsorch.cpp

```diff
@@ -283,14 +258,30 @@ bool PortsOrch::getPort(sai_object_id_t id, Port &port)
     return false;
 }
 
+bool PortsOrch::getPortByBridgePortId(sai_object_id_t bridge_port_id, Port &port)
+{
+    SWSS_LOG_ENTER();
+
+    for (auto &it: m_portList)
+    {
+        if (it.second.m_bridge_port_id == bridge_port_id)
+        {
+            port = it.second;
+            return true;
+        }
+    }
+
+    return false;
+}
+
 void PortsOrch::setPort(string alias, Port p)
 {
     m_portList[alias] = p;
 }
 
-sai_object_id_t PortsOrch::getCpuPort()
+void PortsOrch::getCpuPort(Port &port)
```

Comment:
i want to align with the rest of the getPort functions as the same format.

---
In reply to: [127010996](https://github.com/Azure/sonic-swss/pull/250#discussion_r127010996) [](ancestors = 127010996)